src/EnergyPlus/DaylightingManager.cc

```diff
@@ -7358,18 +7357,18 @@ void DayltgInterReflectedIllum(EnergyPlusData &state,
     // I = sky type;
     // J = 1 for bare window, 2 and above for window with shade or blind.
     auto &ZSK = state.dataDaylightingManager->ZSK;
-    auto &U = state.dataDaylightingManager->U;
-    auto &DayltgInterReflectedIllumNearestHitPt = state.dataDaylightingManager->DayltgInterReflectedIllumNearestHitPt;
-    auto &DayltgInterReflectedIllumObsHitPt = state.dataDaylightingManager->DayltgInterReflectedIllumObsHitPt;
-    auto &DayltgInterReflectedIllumGroundHitPt = state.dataDaylightingManager->DayltgInterReflectedIllumGroundHitPt;
+    Vector3<Real64> U; // Unit vector in (PH,TH) direction
+    Vector3<Real64> nearestHitPt; // Hit point of ray on nearest obstruction (m)
+    Vector3<Real64> obsHitPt; // Coordinates of hit point on an obstruction (m)
+    Vector3<Real64> groundHitPt; // Coordinates of point that ray from window center hits the ground (m)
     auto &SkyObstructionMult = state.dataDaylightingManager->SkyObstructionMult;
```

Comment:
May be possible to make these local as well, but it wasn't immediately obvious. Will look again in the next pass.